src/documents/timecard.md

```diff
@@ -0,0 +1,55 @@
+# Timecard
+
+
+The timecard page enables the user to record their work times via a sequence of screens. Currently the only developed timecardPeriodType is Shift. After choosing this radio button option, the user is taken to the TimecardEntriesList component where they can input the times and dates worked:
+
+![Alt text](images/Screenshot%202023-06-22%20at%2008.15.32.png)
+
+
+After clicking save, the screen displays the recorded shift with the option to remove or edit it: 
+
+![Alt text](images/Screenshot%202023-06-22%20at%2008.46.42.png)
+
+When editing the saved time, the function 'isFinishTimeOnNextDay' calculates whether the time is on the next day and the UI notifies the user with dynamic text:
```

Comment:
Maybe reword 'isFinishTimeOnNextDay' functionality since it works on insert as well as edit.
The main point to get across is the reason why it's there, so users on night shift can come on and quickly type in  their hours without having to manually set the dates.